docs/web/authentication.md

```diff
@@ -219,6 +219,15 @@ servers as it can elongate the authentication process.
 
    Example configuration: `(&(objectClass=person)(sAMAccountName=$USN$))`
 
+ * `user_dn_postfix_preference`
+
+    User DN postfix preference value can be used to select out one prefered
+    user DN if multiple DN entries are found buy the LDAP search.
+    The configured value will be matched and the first matching will be used.
+    If only one DN was found this postfix mathcing will not be used.
+
+   Example configuration: `OU=people,DC=example,DC=com`
```

Comment:
Could you please describe the default behavior when this value is null in the config file or when this option cannot be found in this file (for example I am using an old server config file).